pom.xml

```diff
@@ -1,10 +1,10 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" 
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.committed</groupId>
   <artifactId>speedy-spotless-maven-plugin</artifactId>
   <packaging>maven-plugin</packaging>
-  <version>0.1.0</version>
+  <version>0.1.1</version>
```

Comment:
This needs to use an earlier version of speedy-spotless because 0.1.1 wont exist yet.